src/index.css

```diff
@@ -0,0 +1,4 @@
+#root {
```

Comment:
- [ ] IDs shouldn't be used in selectors because these rules are too tightly coupled with the HTML and have no possibility of reuse.